actors/builtin/storage_power/storage_power_actor.go

```diff
@@ -58,7 +58,7 @@ type AddBalanceParams struct {
 }
 
 func (a *StoragePowerActor) AddBalance(rt Runtime, params *AddBalanceParams) *adt.EmptyValue {
-	builtin.RT_MinerEntry_ValidateCaller_DetermineFundsLocation(rt, params.miner, builtin.MinerEntrySpec_MinerOnly)
+	builtin.PledgeAddress(rt, params.miner)
```

Comment:
Since the side-effect of verification is an important part of this, rename it to `ValidatePledgeAddress`